queries/highlights.scm

```diff
@@ -140,6 +155,13 @@
       right: (identifier) @variable))
   (#match? @keyword "^(def|defdelegate|defguard|defguardp|defmacro|defmacrop|defn|defnp|defp)$"))
 
+; * pipe into field without parentheses (function call)
+(binary_operator
+  operator: "|>"
+  right: (call
+    target: (dot
+      right: (identifier) @function)))
```

Comment:
For what is worth, Elixir will likely emit runtime warnings for this in the future (i.e. Elixir agrees it is a function call but it will treat the code without parens as deprecated).

Reply:
You CALL!

Reply:
Do you mean the same for `foo |> bar`?

Reply:
We will warn on `foo |> bar` in the future as well. But first we want to change the formatter to automatically add the parens.

Reply:
Sounds good. I think it's fine to keep highlighting for now and align once is changes upstream. Highlighting rules are an easy change.

Reply:
Yes, to be clear, it will always be a call. So this is correct. We will just emit warnings too.